src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -79,7 +79,7 @@ protected string ParseServiceName(ConfigurationKeyValue kv)
 
 			if (string.IsNullOrEmpty(retVal))
 			{
-				var stackFrames = new StackTrace().GetFrames();
+				var stackFrames = new StackTrace(true).GetFrames();
```

Comment:
This one here is just for the calculating the service name in case `Assembly.GetEntryAssembly()` returns `null`, so we don't use the line number info here. Could you please remove this one to save overhead?

Reply:
Hi @gregkalapos done 👍